LayoutTests/fast/table/overflowScroll-display-block-expected.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<style>
+div
+{
+    margin-bottom: 35px;
+    width: 100px;
+    height: 100px;
+    background-color: green;
+}
+</style>
+<div style="overflow: scroll;">
+</div>
+<div style="overflow: scroll;">
+</div>
+<div style="overflow: scroll;">
+</div>
+<p> crbug.com/368699: When table elements have display: block, they should be allowed to have overflow:scroll. The exception is the table element, which doesn't permit anything other than visible per http://dev.w3.org/csswg/css2/visufx.html#overflow.</p>
```

Comment:
It would be nice to make WPT versions of these old Chrome tests.

Reply:
Good idea! Let me try it later today. If not, I will push this one and deal with tests directly on WPT and then add them here.

Reply:
Part 1 to sync easy one to WPT - PR raised. Will do others later. Closing this right now and merging.